corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/TokenizationGenerator.java

```diff
@@ -0,0 +1,86 @@
+package edu.illinois.cs.cogcomp.nlp.corpusreaders;
+
+import java.io.File;
+import java.io.IOException;
+
+import edu.illinois.cs.cogcomp.annotation.TextAnnotationBuilder;
+import edu.illinois.cs.cogcomp.core.datastructures.ViewNames;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.Constituent;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotation;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.View;
+import edu.illinois.cs.cogcomp.core.io.LineIO;
+import edu.illinois.cs.cogcomp.core.utilities.TextCleaner;
+import edu.illinois.cs.cogcomp.nlp.corpusreaders.ereReader.CoNLL2002Writer;
+import edu.illinois.cs.cogcomp.nlp.tokenizer.IllinoisTokenizer;
+import edu.illinois.cs.cogcomp.nlp.tokenizer.StatefulTokenizer;
+import edu.illinois.cs.cogcomp.nlp.utility.TokenizerTextAnnotationBuilder;
+
+public class TokenizationGenerator {
+    /**
+     * Just for testing.
+     * 
+     * @param args
+     * @throws IOException 
+     */
```

Comment:
I think I will remove this as well, this was used to evaluate the IllinoisTokenizer against the stageful tokenizer to make a final call on which to use. It could easily be reproduced, but in any event, certainly doesn't belong in this package.